integration-cli/docker_api_containers_test.go

```diff
@@ -879,11 +879,14 @@ func (s *DockerSuite) TestContainerApiBadPort(c *check.C) {
 	config := map[string]interface{}{
 		"Image": "busybox",
 		"Cmd":   []string{"/bin/sh", "-c", "echo test"},
-		"PortBindings": map[string]interface{}{
-			"8080/tcp": []map[string]interface{}{
-				{
-					"HostIP":   "",
-					"HostPort": "aa80",
+		"HostConfig": map[string]interface{}{
+			"NetworkMode": "default",
```

Comment:
@jhowardmsft I understand your point. But the question was more open than just the IT test-case. Will it impact existing apps that assume (correctly or incorrectly) that the an empty net-mode will be "bridge" ? and is that broken with this change ?
Also, in https://github.com/docker/docker/blob/master/daemon/container_unix.go#L918, we allow for an empty networkmode (just for such cases of existing application assuming empty networkmode means it is operating in default/bridge mode).
Hence, IMHO, any change that impacts backward compatibility must be clearly highlighted and have a wider understanding.